json/soknad/bostotte/bostotteutbetaling.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "object",
+  "title": "Bostøtteutbetalinger søker har mottatt",
+  "javaType": "no.nav.sbl.soknadsosialhjelp.soknad.bostotte.BostotteUtbetaling",
+  "properties": {
+    "kilde": {
+      "$ref": "../parts/kildeSystem.json"
+    },
+    "type": {
+      "description": "Kodeverdi som angir hva slags type utbetaling det er. Dette feltet kan for eksempel brukes til å filtrere bort utbetalinger man ikke ønsker å vise til saksbehandler. Det er et MÅ-krav for konsumenter å dynamisk støtte nye typer. Eksempler: \"utbytte\", \"salg\", \"forsikring\" og \"annen\".",
+      "type": "string"
+    },
+    "tittel": {
+      "description": "En tittel som MÅ brukes hvis utbetalingen skal presenteres til saksbehandler. Brukerangitt tekst kan være inkludert i tittelen.",
```

Comment:
"Brukerangitt tekst kan være inkludert i tittelen." - Kan det det for bostotteutbetaling eller er det bare kopiert fra utbetaling.json?